diff --git a/chrome/installer/mini_installer/BUILD.gn b/chrome/installer/mini_installer/BUILD.gn
--- a/chrome/installer/mini_installer/BUILD.gn
+++ b/chrome/installer/mini_installer/BUILD.gn
@@ -5,5 +5,6 @@
 import("//build/config/compiler/compiler.gni")
 import("//build/config/python.gni")
 import("//build/config/ui.gni")
+import("//build/config/chrome_build.gni")
 import("//build/timestamp.gni")
 import("//chrome/process_version_rc_template.gni")
@@ -143,11 +144,11 @@
   release_file = "chrome.release"

   inputs = [
     "$root_out_dir/chrome.dll",
     "$root_out_dir/chrome_elf.dll",
-    "$root_out_dir/chrome.exe",
+    "$root_out_dir/${chrome_executable_name}.exe",
     "$root_out_dir/locales/en-US.pak",
     "$root_out_dir/setup.exe",
     "//chrome/tools/build/win/makecab.py",
     release_file,
   ]
