diff --git a/build/win/reorder-imports.py b/build/win/reorder-imports.py
--- a/build/win/reorder-imports.py
+++ b/build/win/reorder-imports.py
@@ -22,6 +22,6 @@ def reorder_imports(input_dir, output_dir, architecture, exe_name='chrome'):
   # only needs to verify that and not write a whole new exe.
 
-  input_image = os.path.join(input_dir, 'chrome.exe')
-  output_image = os.path.join(output_dir, 'chrome.exe')
+  input_image = os.path.join(input_dir, exe_name + '.exe')
+  output_image = os.path.join(output_dir, exe_name + '.exe')
 
   # pefile mmap()s the whole executable, and then parses parts of
@@ -72,7 +72,7 @@ def reorder_imports(input_dir, output_dir, architecture, exe_name='chrome'):
 
   pe.write(filename=output_image)
 
-  for fname in glob.iglob(os.path.join(input_dir, 'chrome.exe.*')):
+  for fname in glob.iglob(os.path.join(input_dir, exe_name + '.exe.*')):
     shutil.copy(fname, os.path.join(output_dir, os.path.basename(fname)))
   return 0
 
@@ -78,6 +78,8 @@ def reorder_imports(input_dir, output_dir, architecture, exe_name='chrome'):
 def main(argv):
   usage = 'reorder_imports.py -i <input_dir> -o <output_dir> -a <target_arch>'
   parser = optparse.OptionParser(usage=usage)
+  parser.add_option('-e', '--exe', help='executable base name (default: chrome)',
+      metavar='NAME', default='chrome')
   parser.add_option('-i', '--input', help='reorder chrome.exe in DIR',
       metavar='DIR')
   parser.add_option('-o', '--output', help='write new chrome.exe to DIR',
@@ -88,5 +90,5 @@ def main(argv):
 
   if not opts.input or not opts.output:
     parser.error('Please provide and input and output directory')
-  return reorder_imports(opts.input, opts.output, opts.arch)
+  return reorder_imports(opts.input, opts.output, opts.arch, opts.exe)
 
 if __name__ == "__main__":
diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -98,6 +98,8 @@ if (is_win) {
       rebase_path("$root_out_dir/initialexe", root_build_dir),
       "-o",
       rebase_path("$root_out_dir", root_build_dir),
+      "-e",
+      chrome_executable_name,
       "-a",
       current_cpu,
     ]
