diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -1673,7 +1673,7 @@ if (is_win && enable_resource_allowlist_generation) {
   generate_resource_allowlist("resource_allowlist") {
     deps = [ ":chrome_dll" ]
-    inputs = [ "$root_out_dir/chrome.dll.pdb" ]
+    inputs = [ "$root_out_dir/${chrome_executable_name}.dll.pdb" ]
     output = _chrome_resource_allowlist
   }
 }
