diff --git a/chrome/browser/resources/BUILD.gn b/chrome/browser/resources/BUILD.gn
index 6b377ecf8d48c..ebe3672d3b84d 100644
--- a/chrome/browser/resources/BUILD.gn
+++ b/chrome/browser/resources/BUILD.gn
@@ -337,6 +337,7 @@ group("dev_ui_resources") {
       "infobar_internals:resources",
       "media_router/internals:resources",
       "on_device_internals:resources",
+      "review:resources",
       "user_education_internals:resources",
       "webui_js_error:resources",
     ]
@@ -411,6 +412,7 @@ repack("dev_ui_paks") {
       "$root_gen_dir/chrome/infobar_internals_resources.pak",
       "$root_gen_dir/chrome/media_router_internals_resources.pak",
       "$root_gen_dir/chrome/on_device_internals_resources.pak",
+      "$root_gen_dir/chrome/review_resources.pak",
       "$root_gen_dir/chrome/user_education_internals_resources.pak",
       "$root_gen_dir/chrome/webui_js_error_resources.pak",
     ]
