diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 687e7968ad974..2f56cdedad4bc 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -3325,6 +3325,7 @@ static_library("ui") {
 
     deps += [
       "//chrome/browser:titlebar_config",
+      "//chrome/browser/simprint",
       "//chrome/browser/win:mica_titlebar",
       "//chrome/browser/win/conflicts:module_info",
       "//chrome/credential_provider/common:common_constants",
