diff --git a/chrome/common/chrome_switches.h b/chrome/common/chrome_switches.h
index 3acc0029265b0..40a8a7c24ec53 100644
--- a/chrome/common/chrome_switches.h
+++ b/chrome/common/chrome_switches.h
@@ -284,6 +284,8 @@ extern const char kSourceAppId[];
 extern const char kSourceShortcut[];
 extern const char kUninstall[];
 extern const char kUninstallAppId[];
+extern const char kSimprintEnvId[];  // Simprint environment ID (UUID)
+extern const char kSimprintDisplayId[];  // Simprint display ID (numeric) for taskbar badge
 #endif  // BUILDFLAG(IS_WIN)
 
 #if BUILDFLAG(ENABLE_PRINT_PREVIEW) && !defined(OFFICIAL_BUILD)
