diff --git a/chrome/common/webui_url_constants.cc b/chrome/common/webui_url_constants.cc
index f96fccfbc248a..4f3905d11e62c 100644
--- a/chrome/common/webui_url_constants.cc
+++ b/chrome/common/webui_url_constants.cc
@@ -124,6 +124,7 @@ base::span<const base::cstring_view> ChromeURLHosts() {
 #if !BUILDFLAG(IS_ANDROID)
       kChromeUIReloadButtonHost,
 #endif
+      kChromeUIReviewHost,
       kChromeUISignInInternalsHost,
       kChromeUISiteEngagementHost,
 #if !BUILDFLAG(IS_ANDROID)
