diff --git a/net/socket/connect_job_params_factory.cc b/net/socket/connect_job_params_factory.cc
index e975e09a16b84..1c3504b2dd9f5 100644
--- a/net/socket/connect_job_params_factory.cc
+++ b/net/socket/connect_job_params_factory.cc
@@ -26,6 +26,7 @@
 #include "net/socket/next_proto.h"
 #include "net/socket/socket_tag.h"
 #include "net/socket/socks_connect_job.h"
+#include "net/socket/socks5_auth_cache.h"
 #include "net/socket/ssl_connect_job.h"
 #include "net/socket/transport_connect_job.h"
 #include "net/ssl/ssl_config.h"
@@ -262,11 +263,17 @@ ConnectJobParams CreateProxyParams(
   } else {
     DCHECK(proxy_server.is_socks());
     DCHECK_EQ(1u, proxy_chain.length());
+    std::optional<AuthCredentials> socks5_credentials;
+    if (proxy_server.scheme() == ProxyServer::SCHEME_SOCKS5) {
+      socks5_credentials = SOCKS5AuthCache::GetInstance().GetCredentials(
+          HostPortPair(proxy_server.GetHost(), proxy_server.GetPort()));
+    }
     // TODO(crbug.com/40181080): Pass `endpoint` directly (preserving scheme
     // when available)?
     params = ConnectJobParams(base::MakeRefCounted<SOCKSSocketParams>(
         std::move(params), proxy_server.scheme() == ProxyServer::SCHEME_SOCKS5,
-        ToHostPortPair(endpoint), network_anonymization_key,
+        ToHostPortPair(endpoint), std::move(socks5_credentials),
+        network_anonymization_key,
         *proxy_annotation_tag));
   }
 
