diff --git a/net/socket/socks_connect_job.h b/net/socket/socks_connect_job.h
index 7dc53ec895d49..1972ea999f4e2 100644
--- a/net/socket/socks_connect_job.h
+++ b/net/socket/socks_connect_job.h
@@ -6,6 +6,7 @@
 #define NET_SOCKET_SOCKS_CONNECT_JOB_H_
 
 #include <memory>
+#include <optional>
 #include <set>
 
 #include "base/memory/raw_ptr.h"
@@ -15,6 +16,7 @@
 #include "net/base/host_port_pair.h"
 #include "net/base/net_errors.h"
 #include "net/base/net_export.h"
+#include "net/base/auth.h"
 #include "net/base/network_isolation_key.h"
 #include "net/base/request_priority.h"
 #include "net/dns/public/resolve_error_info.h"
@@ -35,6 +37,7 @@ class NET_EXPORT_PRIVATE SOCKSSocketParams
   SOCKSSocketParams(ConnectJobParams nested_params,
                     bool socks_v5,
                     const HostPortPair& host_port_pair,
+                    std::optional<AuthCredentials> socks5_credentials,
                     const NetworkAnonymizationKey& network_anonymization_key,
                     const NetworkTrafficAnnotationTag& traffic_annotation);
 
@@ -46,6 +49,9 @@ class NET_EXPORT_PRIVATE SOCKSSocketParams
   }
   const HostPortPair& destination() const { return destination_; }
   bool is_socks_v5() const { return socks_v5_; }
+  const std::optional<AuthCredentials>& socks5_credentials() const {
+    return socks5_credentials_;
+  }
   const NetworkAnonymizationKey& network_anonymization_key() {
     return network_anonymization_key_;
   }
@@ -63,6 +69,7 @@ class NET_EXPORT_PRIVATE SOCKSSocketParams
   // This is the HTTP destination.
   const HostPortPair destination_;
   const bool socks_v5_;
+  const std::optional<AuthCredentials> socks5_credentials_;
   const NetworkAnonymizationKey network_anonymization_key_;
 
   NetworkTrafficAnnotationTag traffic_annotation_;
