diff --git a/services/network/network_context.cc b/services/network/network_context.cc
index cd39392504a0f..255edd8f776b4 100644
--- a/services/network/network_context.cc
+++ b/services/network/network_context.cc
@@ -60,6 +60,7 @@
 #include "components/url_pattern/simple_url_pattern_matcher.h"
 #include "mojo/public/cpp/bindings/pending_receiver.h"
 #include "net/base/features.h"
+#include "net/base/host_port_pair.h"
 #include "net/base/isolation_info.h"
 #include "net/base/load_flags.h"
 #include "net/base/net_errors.h"
@@ -98,6 +99,7 @@
 #include "net/proxy_resolution/configured_proxy_resolution_service.h"
 #include "net/proxy_resolution/proxy_config.h"
 #include "net/shared_dictionary/shared_dictionary_isolation_key.h"
+#include "net/socket/socks5_auth_cache.h"
 #include "net/ssl/ssl_config_service.h"
 #include "net/storage_access_api/status.h"
 #include "net/traffic_annotation/network_traffic_annotation.h"
@@ -795,6 +797,16 @@ NetworkContext::NetworkContext(
             ->allow_default_credentials);
   }
 
+  if (!params_->socks5_auth_entries.empty()) {
+    net::SOCKS5AuthCache::GetInstance().ClearCredentials();
+    for (const auto& socks5_auth_entry : params_->socks5_auth_entries) {
+      net::SOCKS5AuthCache::GetInstance().SetCredentials(
+          net::HostPortPair(socks5_auth_entry->host, socks5_auth_entry->port),
+          net::AuthCredentials(base::UTF8ToUTF16(socks5_auth_entry->username),
+                               base::UTF8ToUTF16(socks5_auth_entry->password)));
+    }
+  }
+
   InitializeCorsParams();
 
 #if BUILDFLAG(IS_CT_SUPPORTED)
