diff --git a/services/network/network_service.h b/services/network/network_service.h
index d3933748841cf..6ab02b4a7f142 100644
--- a/services/network/network_service.h
+++ b/services/network/network_service.h
@@ -175,6 +175,10 @@ class COMPONENT_EXPORT(NETWORK_SERVICE) NetworkService
       mojom::HttpAuthStaticParamsPtr http_auth_static_params) override;
   void ConfigureHttpAuthPrefs(
       mojom::HttpAuthDynamicParamsPtr http_auth_dynamic_params) override;
+  void SetSocks5AuthEntries(
+      std::vector<mojom::Socks5AuthEntryPtr> socks5_auth_entries) override;
+  void CloseAllConnections() override;
+  void ClearHttpAuthCache() override;
   void SetRawHeadersAccess(int32_t process_id,
                            const std::vector<url::Origin>& origins) override;
   void SetMaxConnectionsPerProxyChain(uint32_t max_connections) override;
