diff --git a/services/network/public/mojom/network_context.mojom b/services/network/public/mojom/network_context.mojom
index 709aae5568b73..ffc866085018c 100644
--- a/services/network/public/mojom/network_context.mojom
+++ b/services/network/public/mojom/network_context.mojom
@@ -198,6 +198,13 @@ struct HttpAuthStaticNetworkContextParams {
       DefaultCredentials.ALLOW_DEFAULT_CREDENTIALS;
 };
 
+struct Socks5AuthEntry {
+  string host;
+  uint16 port;
+  string username;
+  string password;
+};
+
 struct NetworkContextFilePaths {
   // Points to the shared dictionary directory containing the sqlite database
   // file named "db" for metadata and the disk cache directory named "cache".
@@ -510,6 +517,10 @@ struct NetworkContextParams {
   // HttpAuthPreference.
   HttpAuthStaticNetworkContextParams? http_auth_static_network_context_params;
 
+  // SOCKS5 username/password credentials that must be available in the
+  // network service process when constructing SOCKS5 client sockets.
+  array<Socks5AuthEntry> socks5_auth_entries;
+
   // Indicates that the HttpCache back-end should be reset (e.g. when a Finch
   // experiment changes the cache key representation via NetworkIsolationKey).
   // A reset throws out all existing cache content when loading the cache,
