diff --git a/chrome/browser/resources/new_tab_page/BUILD.gn b/chrome/browser/resources/new_tab_page/BUILD.gn
index 7b86387cecf18..641523ee8eaf2 100644
--- a/chrome/browser/resources/new_tab_page/BUILD.gn
+++ b/chrome/browser/resources/new_tab_page/BUILD.gn
@@ -57,6 +57,7 @@ build_webui("build") {
     "//chrome/browser/ui/webui/new_tab_page:mojo_bindings_ts__generator",
     "//chrome/browser/ui/webui/new_tab_page/action_chips:mojo_bindings_ts__generator",
     "//chrome/browser/ui/webui/new_tab_page/ntp_promo:mojo_bindings_ts__generator",
+    "//chrome/browser/ui/webui/new_tab_page/simprint_ip:mojo_bindings_ts__generator",
     "//chrome/browser/ui/webui/ntp_microsoft_auth:shared_ts__generator",
     "//chrome/browser/ui/webui/side_panel/customize_chrome:mojo_bindings_ts__generator",
     "//components/ntp_tiles:mojo_bindings_ts__generator",
@@ -77,6 +78,7 @@ build_webui("build") {
     "$root_gen_dir/chrome/browser/new_tab_page/modules/v2/tab_groups/tab_groups.mojom-webui.ts",
     "$root_gen_dir/chrome/browser/ui/webui/new_tab_page/action_chips/action_chips.mojom-webui.ts",
     "$root_gen_dir/chrome/browser/ui/webui/new_tab_page/new_tab_page.mojom-webui.ts",
+    "$root_gen_dir/chrome/browser/ui/webui/new_tab_page/simprint_ip/simprint_ip.mojom-webui.ts",
     "$root_gen_dir/chrome/browser/ui/webui/ntp_microsoft_auth/ntp_microsoft_auth_shared_ui.mojom-webui.ts",
     "$root_gen_dir/chrome/browser/ui/webui/customize_buttons/customize_buttons.mojom-webui.ts",
     "$root_gen_dir/ui/webui/resources/cr_components/composebox/composebox.mojom-webui.ts",
