diff --git a/chrome/browser/resources/new_tab_page/new_tab_page.gni b/chrome/browser/resources/new_tab_page/new_tab_page.gni
index f68a6c9f7f368..aaa1ac55db3c0 100644
--- a/chrome/browser/resources/new_tab_page/new_tab_page.gni
+++ b/chrome/browser/resources/new_tab_page/new_tab_page.gni
@@ -18,6 +18,7 @@ all_ts_files = [
                  "metrics_utils.ts",
                  "new_tab_page_proxy.ts",
                  "new_tab_page.ts",
+                 "simprint_ntp.ts",
                  "transparency.ts",
                  "utils.ts",
                  "window_proxy.ts",
