diff --git a/chrome/browser/resources/new_tab_page/new_tab_page.gni b/chrome/browser/resources/new_tab_page/new_tab_page.gni
index f68a6c9f7f368..aaa1ac55db3c0 100644
--- a/chrome/browser/resources/new_tab_page/new_tab_page.gni
+++ b/chrome/browser/resources/new_tab_page/new_tab_page.gni
@@ -18,6 +18,8 @@ all_ts_files = [
                  "metrics_utils.ts",
                  "new_tab_page_proxy.ts",
                  "new_tab_page.ts",
+                 "simprint_ntp.ts",
+                 "simprint_ip_proxy.ts",
                  "transparency.ts",
                  "utils.ts",
                  "window_proxy.ts",
@@ -64,6 +66,7 @@ if (optimize_webui) {
     "new_tab_page.mojom-webui.js",
     "ntp_microsoft_auth_shared_ui.mojom-webui.js",
     "outlook_calendar.mojom-webui.js",
+    "simprint_ip.mojom-webui.js",
     "tab_groups.mojom-webui.js",
     "tab_group_types.mojom-webui.js",
     "tile_type.mojom-webui.js",
