diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 6fa22a1e7ef1e..a2523f9c10bf6 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1190,6 +1190,8 @@ static_library("ui") {
       "webui/new_tab_page/new_tab_page_ui.h",
       "webui/new_tab_page/ntp_pref_names.cc",
       "webui/new_tab_page/ntp_pref_names.h",
+      "webui/new_tab_page/simprint_ip/simprint_ip_handler.cc",
+      "webui/new_tab_page/simprint_ip/simprint_ip_handler.h",
       "webui/new_tab_page/untrusted_source.cc",
       "webui/new_tab_page/untrusted_source.h",
       "webui/new_tab_page_third_party/new_tab_page_third_party_handler.cc",
