diff --git a/chrome/browser/chrome_browser_main.cc b/chrome/browser/chrome_browser_main.cc
index d347c428acc41..9011d873638c3 100644
--- a/chrome/browser/chrome_browser_main.cc
+++ b/chrome/browser/chrome_browser_main.cc
@@ -137,6 +137,10 @@
 #include "ui/color/color_provider_manager.h"
 #include "ui/gl/gl_switches.h"
 
+#if BUILDFLAG(IS_WIN)
+#include "simprint/eventbus/eventbus.h"
+#endif
+
 // Per-platform #include blocks, in alphabetical order.
 
 #if BUILDFLAG(IS_ANDROID)
@@ -1566,6 +1570,19 @@ void ChromeBrowserMainParts::PostBrowserStart() {
 
   browser_process_->browser_policy_connector()->OnBrowserStarted();
 
+#if BUILDFLAG(IS_WIN)
+  // Initialize Simprint EventBus if env_id is provided
+  {
+    const base::CommandLine* command_line =
+        base::CommandLine::ForCurrentProcess();
+    if (command_line->HasSwitch("simprint-env-id")) {
+      std::string env_id =
+          command_line->GetSwitchValueASCII("simprint-env-id");
+      simprint::eventbus::EventBus::GetInstance().Initialize(env_id);
+    }
+  }
+#endif
+
 #if BUILDFLAG(ENABLE_PROCESS_SINGLETON)
   // Allow ProcessSingleton to process messages.
   // This is done here instead of just relying on the main message loop's start
@@ -2033,10 +2050,15 @@ void ChromeBrowserMainParts::PostMainMessageLoopRun() {
   TRACE_EVENT_END("toplevel", perfetto::Track::FromPointer(this));
   TRACE_EVENT0("startup", "ChromeBrowserMainParts::PostMainMessageLoopRun");
 #if BUILDFLAG(IS_ANDROID)
-  // Chrome on Android does not use default MessageLoop. It has its own
-  // Android specific MessageLoop
+  // On Android, there is no quit/exit. So the browser's main message loop will
+  // not finish.
   NOTREACHED();
 #else
+#if BUILDFLAG(IS_WIN)
+  // Shutdown Simprint EventBus
+  simprint::eventbus::EventBus::GetInstance().Shutdown();
+#endif
+
   // Shutdown the UpgradeDetector here before `ChromeBrowserMainPartsAsh`
   // disconnects DBus services in its PostDestroyThreads.
   UpgradeDetector::GetInstance()->Shutdown();
