diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index a2523f9c10bf6..bd4e3d7718b97 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -327,6 +327,8 @@ static_library("ui") {
     "//chrome/browser/search_engines",
     "//chrome/browser/segmentation_platform",
     "//chrome/browser/sessions",
+    "//simprint/console_log",
+    "//simprint/eventbus",
     "//chrome/browser/share",
     "//chrome/browser/signin",
     "//chrome/browser/signin:identity_manager_provider",
@@ -3900,6 +3902,10 @@ static_library("ui") {
       "views/frame/scrim_view_controller.h",
       "views/frame/shadow_overlay_view.cc",
       "views/frame/shadow_overlay_view.h",
+      "views/frame/sync_input_capture_handler.cc",
+      "views/frame/sync_input_capture_handler.h",
+      "views/frame/sync_input_replay.cc",
+      "views/frame/sync_input_replay.h",
       "views/frame/system_menu_model_builder.cc",
       "views/frame/system_menu_model_builder.h",
       "views/frame/system_menu_model_delegate.cc",
