diff --git a/chrome/browser/ui/views/chrome_views_delegate.cc b/chrome/browser/ui/views/chrome_views_delegate.cc
index d9c2cf2a377b7..9d58b2c2713b4 100644
--- a/chrome/browser/ui/views/chrome_views_delegate.cc
+++ b/chrome/browser/ui/views/chrome_views_delegate.cc
@@ -7,6 +7,7 @@
 #include <memory>
 
 #include "base/check_op.h"
+#include "base/no_destructor.h"
 #include "build/build_config.h"
 #include "chrome/browser/browser_process.h"
 #include "chrome/browser/profiles/keep_alive/profile_keep_alive_types.h"
@@ -222,6 +223,30 @@ void ChromeViewsDelegate::OnBeforeWidgetInit(
   params->native_widget = CreateNativeWidget(params, delegate);
 }
 
+namespace {
+
+ChromeViewsDelegate::SyncInputOnWidgetInitializedCallback*
+GetSyncInputOnWidgetInitializedStorage() {
+  static base::NoDestructor<
+      ChromeViewsDelegate::SyncInputOnWidgetInitializedCallback>
+      storage;
+  return storage.operator->();
+}
+
+}  // namespace
+
+void ChromeViewsDelegate::SetSyncInputOnWidgetInitialized(
+    SyncInputOnWidgetInitializedCallback callback) {
+  *GetSyncInputOnWidgetInitializedStorage() = std::move(callback);
+}
+
+void ChromeViewsDelegate::OnWidgetInitialized(views::Widget* widget) {
+  auto* cb = GetSyncInputOnWidgetInitializedStorage();
+  if (cb && !cb->is_null()) {
+    cb->Run(widget);
+  }
+}
+
 std::string ChromeViewsDelegate::GetApplicationName() {
   return std::string(version_info::GetProductName());
 }
