diff --git a/ui/views/views_delegate.h b/ui/views/views_delegate.h
index e335afe969ca1..1116d863afddd 100644
--- a/ui/views/views_delegate.h
+++ b/ui/views/views_delegate.h
@@ -166,6 +166,10 @@ class VIEWS_EXPORT ViewsDelegate {
   virtual void OnBeforeWidgetInit(Widget::InitParams* params,
                                   internal::NativeWidgetDelegate* delegate);
 
+  // Called when a Widget has finished Init (after native widget is created).
+  // Embedders can use this to e.g. attach input handlers to popup roots.
+  virtual void OnWidgetInitialized(Widget* widget) {}
+
   // Returns true if the operating system's window manager will always provide a
   // title bar with caption buttons (ignoring the setting to
   // |remove_standard_frame| in InitParams). If |maximized|, this applies to
