diff --git a/ui/views/widget/widget.cc b/ui/views/widget/widget.cc
index c440587111703..29785f04e931b 100644
--- a/ui/views/widget/widget.cc
+++ b/ui/views/widget/widget.cc
@@ -546,6 +546,9 @@ void Widget::Init(InitParams params) {
   if (type == InitParams::TYPE_MENU) {
     is_mouse_button_pressed_ = native_widget_->IsMouseButtonDown();
   }
+  if (ViewsDelegate::GetInstance()) {
+    ViewsDelegate::GetInstance()->OnWidgetInitialized(this);
+  }
   if (RequiresNonClientView(type)) {
     non_client_view_ =
         new NonClientView(widget_delegate_->CreateClientView(this));
