diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
index 900df063b86fa..bc78819660e36 100644
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -15,6 +15,8 @@
 #include <utility>
 #include <vector>
 
+#include "simprint/config/fingerprint_config_storage.h"
+
 #include "base/base_switches.h"
 #include "base/byte_count.h"
 #include "base/check_deref.h"
