diff --git a/chrome/browser/net/profile_network_context_service.h b/chrome/browser/net/profile_network_context_service.h
index f1c7291eca17f..ae756aa71932a 100644
--- a/chrome/browser/net/profile_network_context_service.h
+++ b/chrome/browser/net/profile_network_context_service.h
@@ -141,6 +141,10 @@ class ProfileNetworkContextService
   // Get platform ClientCertStore. May return nullptr.
   std::unique_ptr<net::ClientCertStore> CreateClientCertStore();
 
+  // Simprint: Make UpdateAcceptLanguage public so it can be called when
+  // fingerprint config is loaded
+  void UpdateAcceptLanguage();
+
 #if BUILDFLAG(IS_CHROMEOS)
   // Returns a factory callback that may be run to get the issuer sources for
   // client cert pathbuilding. The factory callback may run its result callback
@@ -162,10 +166,6 @@ class ProfileNetworkContextService
   // Checks |quic_allowed_|, and disables QUIC if needed.
   void DisableQuicIfNotAllowed();
 
-  // Forwards changes to |pref_accept_language_| to the NetworkContext, after
-  // formatting them as appropriate.
-  void UpdateAcceptLanguage();
-
   // Computes appropriate value of Accept-Language header based on
   // |pref_accept_language_|
   std::string ComputeAcceptLanguage() const;
