diff --git a/components/embedder_support/BUILD.gn b/components/embedder_support/BUILD.gn
index 660aeb47d9b12..2add649164aff 100644
--- a/components/embedder_support/BUILD.gn
+++ b/components/embedder_support/BUILD.gn
@@ -63,6 +63,7 @@ static_library("user_agent") {
     "//components/prefs",
     "//components/version_info",
     "//net",
+    "//simprint/config",
     "//third_party/blink/public/common:headers",
     "//ui/base",
   ]
