diff --git a/content/common/renderer.mojom b/content/common/renderer.mojom
index 5442148a11b10..f98235b7c28c9 100644
--- a/content/common/renderer.mojom
+++ b/content/common/renderer.mojom
@@ -177,5 +177,6 @@ interface Renderer {
                      array<string> cors_exempt_header_list,
                      blink.mojom.OriginTrialsSettings? origin_trials_settings,
                      blink.mojom.PerformanceTier cpu_performance_tier,
-                     uint64 trace_id);
+                     uint64 trace_id,
+                     string simprint_fingerprint_config);
 };
