diff --git a/content/renderer/render_thread_impl.h b/content/renderer/render_thread_impl.h
index 9a813a14b285f..b3be306da2e44 100644
--- a/content/renderer/render_thread_impl.h
+++ b/content/renderer/render_thread_impl.h
@@ -408,7 +408,8 @@ class CONTENT_EXPORT RenderThreadImpl
       const std::vector<std::string>& cors_exempt_header_list,
       blink::mojom::OriginTrialsSettingsPtr origin_trial_settings,
       blink::mojom::PerformanceTier cpu_performance_tier,
-      uint64_t trace_id) override;
+      uint64_t trace_id,
+      const std::string& simprint_fingerprint_config) override;
   void UpdateScrollbarTheme(
       mojom::UpdateScrollbarThemeParamsPtr params) override;
   void OnSystemColorsChanged(int32_t aqua_color_variant) override;
