diff --git a/third_party/blink/renderer/modules/donottrack/BUILD.gn b/third_party/blink/renderer/modules/donottrack/BUILD.gn
index 4bcbeba9a9cf1..067471335f073 100644
--- a/third_party/blink/renderer/modules/donottrack/BUILD.gn
+++ b/third_party/blink/renderer/modules/donottrack/BUILD.gn
@@ -9,4 +9,7 @@ blink_modules_sources("donottrack") {
     "navigator_do_not_track.cc",
     "navigator_do_not_track.h",
   ]
+  deps = [
+    "//simprint/fingerprint",
+  ]
 }
