diff --git a/third_party/blink/renderer/modules/peerconnection/BUILD.gn b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
index f4c4410182f08..258169b1e1472 100644
--- a/third_party/blink/renderer/modules/peerconnection/BUILD.gn
+++ b/third_party/blink/renderer/modules/peerconnection/BUILD.gn
@@ -186,6 +186,7 @@ blink_modules_sources("peerconnection") {
     "//third_party/blink/renderer/modules/mediastream",
     "//third_party/blink/renderer/modules/permissions",
     "//third_party/blink/renderer/modules/webrtc",
+    "//simprint/fingerprint",
   ]
 
   allow_circular_includes_from =
