diff --git a/third_party/blink/renderer/modules/speech/BUILD.gn b/third_party/blink/renderer/modules/speech/BUILD.gn
index e49c0234d1651..3423a9b4195d2 100644
--- a/third_party/blink/renderer/modules/speech/BUILD.gn
+++ b/third_party/blink/renderer/modules/speech/BUILD.gn
@@ -43,5 +43,6 @@ blink_modules_sources("speech") {
     "//media",
     "//media/mojo/mojom:speech_recognition_audio_forwarder_blink",
     "//third_party/blink/renderer/modules/mediastream",
+    "//simprint/fingerprint",
   ]
 }
