diff --git a/third_party/blink/renderer/modules/webgpu/BUILD.gn b/third_party/blink/renderer/modules/webgpu/BUILD.gn
index 90b629c55940c..0be7a451577d7 100644
--- a/third_party/blink/renderer/modules/webgpu/BUILD.gn
+++ b/third_party/blink/renderer/modules/webgpu/BUILD.gn
@@ -109,6 +109,7 @@ blink_modules_sources("webgpu") {
   deps = [
     "//gpu/command_buffer/client:webgpu_interface",
     "//services/metrics/public/cpp:ukm_builders",
+    "//simprint/fingerprint",
     "//skia",
     "//skia:skcms",
     "//third_party/blink/renderer/modules/webcodecs",
