diff --git a/third_party/blink/renderer/platform/BUILD.gn b/third_party/blink/renderer/platform/BUILD.gn
index 49d07b1c0cbeb..523f4fb8ad17a 100644
--- a/third_party/blink/renderer/platform/BUILD.gn
+++ b/third_party/blink/renderer/platform/BUILD.gn
@@ -1828,6 +1828,7 @@ component("platform") {
     "//services/network/public/mojom:mojom_shared",
     "//services/service_manager/public/cpp",
     "//services/viz/public/cpp/gpu",
+    "//simprint/config",
     "//skia",
     "//skia:skcms",
     "//third_party/abseil-cpp:absl",
