diff --git a/third_party/blink/renderer/platform/exported/platform.cc b/third_party/blink/renderer/platform/exported/platform.cc
index d76d5096431e2..36d35619f2329 100644
--- a/third_party/blink/renderer/platform/exported/platform.cc
+++ b/third_party/blink/renderer/platform/exported/platform.cc
@@ -180,7 +180,6 @@ void Platform::InitializeMainThreadCommon(
   // In case core is not available (like on PPAPI plugins), we need to init
   // them here.
   font_family_names::Init();
-  InitializePlatformLanguage();
 
   base::trace_event::MemoryDumpManager::GetInstance()->RegisterDumpProvider(
       PartitionAllocMemoryDumpProvider::Instance(), "PartitionAlloc",
