diff --git a/third_party/blink/renderer/platform/language.h b/third_party/blink/renderer/platform/language.h
index 62bd667dcf40f..b3995f501fc81 100644
--- a/third_party/blink/renderer/platform/language.h
+++ b/third_party/blink/renderer/platform/language.h
@@ -44,7 +44,6 @@ PLATFORM_EXPORT void OverrideUserPreferredLanguagesForTesting(
 PLATFORM_EXPORT wtf_size_t
 IndexOfBestMatchingLanguageInList(const AtomicString& language,
                                   const Vector<AtomicString>& language_list);
-PLATFORM_EXPORT void InitializePlatformLanguage();
 
 }  // namespace blink
 
